core/docker/default/etc/jvm.config

```diff
@@ -17,3 +17,5 @@
 -XX:+UseAESCTRIntrinsics
 # Disable Preventive GC for performance reasons (JDK-8293861)
 -XX:-G1UsePreventiveGC
+# Reduce starvation of threads by GClocker, recommend to set about the number of cpu cores (JDK-8192647)
+-XX:GCLockerRetryAllocationCount=32
```

Comment:
> add new line

Done~ I also  added two spaces after these two lines, because .md file needs this to start a new line. That two lines were displayed on single line before.  
![image](https://github.com/trinodb/trino/assets/26461591/1851673a-a246-43e2-82aa-c79309937207)